apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -121,6 +121,28 @@ export function InstructorCourseAdminInstances({
                 </tbody>
               </table>
             </div>
+            ${courseInstances.length === 0
+              ? html`
+                  <div class="my-4">
+                    <p class="text-center font-weight-bold">No course instances found.</p>
+                    <p class="text-center mb-0">
+                      A course instance is a single offering of a course, such as "Fall 2024" or
+                      "Fall 2024, Section M".
+                    </p>
+                    <p class="text-center">
+                      To create one, click <strong>Add course instance</strong>.
+                    </p>
+                    <p class="text-center">
+                      Learn more in the
+                      <a
+                        href="https://prairielearn.readthedocs.io/en/latest/getStarted/#creating-a-course-instance"
```

Comment:
There are two pages that discuss course instances: the one you linked to, and https://prairielearn.readthedocs.io/en/latest/courseInstance/. Any preference which one we should link to? cc @mfsilva22, who probably has thoughts on links from PL to the documentation.

Reply:
I changed the link to this one -- this makes more sense, since the one I had before linked to instructions to create a Course Instance without the modal I'm developing in #11004.

Reply:
Good thinking! Agreed that the new link is better.